spec/readme.md

```diff
@@ -10,7 +10,7 @@ This tar file can be created from most common container tools (e.g., Docker).
 
 ## The base folder
 
-To be a Flywheel gear, the container in the tar file must include a folder named: `/flywheel/v0`.
+To be a Flywheel gear, the container in the tar file must include a folder named: `/flywheel/v0`.<br>
```

Comment:
Having an explicit HTML tag in the markdown seems to go against the markdown intention. Can't we just add a blank line to get the same effect?